brush-shell/tests/cases/builtins/kill.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Builtins: kill"
+cases:
+  - name: "kill -l"
+    ignore_stderr: true
+    # note that we dont use 'kill -l' because of the shape of the bash list
+    # 1) .. 2) .. 3)  .. etc. instead of just a simple list
+    stdin: |
+      # kill -l
+      kill -l 1
+      kill -l 2
+      kill -l 3
+      kill -l 4
+      kill -l 5
+      kill -l 6
+      kill -l 7
+      kill -l 9
```

Comment:
Was `kill -l 8` intentionally omitted? For that matter, what about going with a `for` loop for better maintainability/readability?